scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/models/AutoSkillCommand.kt

```diff
@@ -29,14 +29,38 @@ class AutoSkillCommand private constructor(
             return target
         }
 
+        private fun getTargets(queue: Queue<Char>): List<ServantTarget>? {
+            val targets = ArrayList<ServantTarget>()
+            val nextChar = queue.peek()
+            if (nextChar == '(') {
+                while (true) {
```

Comment:
This is dangerous because if the user manually edits the command (which can be enabled in the options), then they may forget to add a `)` and the queue will run out of elements, leading to a `NoSuchElementException`.

I replaced it with an Exception with a proper error text, but it doesn't have translations since the error shouldn't happen to most users.